visualization/app/codeCharta/ui/ribbonBar/ribbonBar.component.html

```diff
@@ -9,12 +9,16 @@
 	</div>
 </md-card>
 
-<md-card ng-attr-id="area-metric-card">
+<md-card ng-attr-id="area-metric-card" ng-class="{ expanded: $ctrl._viewModel.panelSelection === 'AREA_PANEL_OPEN' }">
 	<div class="section">
```

Comment:
Do you want to set an enum variable `panelSelectionValues: PanelSelection` into the viewModel, so you can replace the string values (also below)? :)

Reply:
Good suggestions. Done.

Another way would be to provide functions from the controller, each checking if a certain panel is open. This would result in n functions, where n is the number of entries in the enum.
All functions implement something like
```ts
public isAreaPanelOpen() {
	return this._viewModel.panelSelection === PanelSelection.AREA_PANEL_OPEN
}
```

Hard to tell which approach is better though